docs/source/examples/classification_and_regression.rst

```diff
@@ -0,0 +1,302 @@
+.. role:: hidden
+    :class: hidden-section
+
+Gender Classification and Eyes Location Detection: A Two Task Problem
+*********************************************************************
+
+.. note::
+
+    - See the notebook `here <https://github.com/GRAAL-Research/poutyne/blob/master/examples/classification_and_regression.ipynb>`_
+    - Run in `Google Colab <https://colab.research.google.com/github/GRAAL-Research/poutyne/blob/master/examples/classification_and_regression.ipynb>`_
+
+In this example, we are going to implement a multi-task problem. We try to identify the gender of the people, as well as locating their eyes in the image. Hence, we have two different tasks: classification (to identify the gender) and regression (to find the location of the eyes). We are going to use a single network (A CNN) to perform both tasks, however, we will need to apply different loss functions, each proper to a specific task.
+
+Let's import all the needed packages.
+
+.. code-block:: python
+
+    import math
+    import os
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import pandas as pd
+    import wget
+    import zipfile
+    import cv2
+    import torch
+    import torch.nn as nn
+    import torch.nn.functional as F
+    import torch.optim as optim
+    from torch.autograd import Variable
```

Comment:
`Variable`s are deprecated and useless in recent versions of PyTorch. Thus, you can remove its uses everywhere and everything will work as expected.

Reply:
Sure. I used it o activate the gradient ( requires_grad=True). So is it also automatically done in the latest version?

Reply:
In the places you used `requires_grad=True`, you don't need to.